profiles/coreos/arm64/package.accept_keywords

```diff
@@ -53,6 +53,8 @@ net-fs/nfs-utils ~arm64
 #=sys-apps/smartmontools-9999 **
 =sys-apps/smartmontools-6.4 **
 
+=app-misc/jq-1.4-r1 **
+=net-misc/curl-7.45.0 ~arm64
```

Comment:
are these not sufficient?

https://github.com/coreos/coreos-overlay/blob/master/profiles/coreos/base/package.accept_keywords#L59
https://github.com/coreos/coreos-overlay/blob/master/profiles/coreos/base/package.accept_keywords#L95